src/chunk/render/tilemap-square.vert

```diff
@@ -45,7 +45,8 @@ void main() {
         vec2(sprite_rect.end.x, sprite_rect.begin.y),
         sprite_rect.end
     );
-    v_Uv = floor(atlas_positions[gl_VertexIndex % 4] + vec2(0.01, 0.01)) / AtlasSize;
+    v_Uv = floor(atlas_positions[gl_VertexIndex % 4]) / AtlasSize;
+    v_Uv += 1e-5;
```

Comment:
This is essentially a super small fractional number that we shift the UV by which helps to clear up the artifacts. I'm not sure if this will fix it in all cases however.